src/userspace/exec_monitor.cpp

```diff
@@ -65,11 +66,31 @@ int ExecMonitor::run()
 		std::cerr << "[EXEC_MONITOR]: Error allocating the ringbufffer.\n";
 		goto out;
 	}
-	
+
+	if (n_proc > 0) {
+		skel->bss->n_monitored_proc = 0;
```

Comment:
Variables in the bss segment should already be zeroed https://en.wikipedia.org/wiki/.bss#BSS_in_C